src/Workspaces/MSBuildTest/Resources/CircularProjectReferences/CircularCSharpProject1.csproj

```
@@ -11,7 +11,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Library1</RootNamespace>
     <AssemblyName>Library1</AssemblyName>
-    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
+   <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
```

```suggestion
    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
```

This missing space is in all the changes I can see.